frontends/ol-components/src/components/Button/Button.tsx

```diff
@@ -196,10 +196,14 @@ const iconButtonDefaultProps: Required<
   color: "primary",
 }
 
-type IconButtonProps = Omit<ButtonStyleProps, "startIcon" | "endIcon"> &
+type ActionButtonProps = Omit<ButtonStyleProps, "startIcon" | "endIcon"> &
   React.ComponentProps<"button">
-const IconButton = styled(ButtonStyled)((props: IconButtonProps) => {
-  const { size = iconButtonDefaultProps.size } = props
+const ActionButton = styled(
+  React.forwardRef<HTMLButtonElement, ActionButtonProps>((props, ref) => (
+    <ButtonStyled ref={ref} type="button" {...props} />
```

Comment:
The missing `type="button"` before was a bug that had gone unnoticed since we accidentally weren't using this component.